ci/forge/Dockerfile.forge

```diff
@@ -31,7 +31,8 @@ curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
 
 apt-get update
 apt-get install -y \
-  awscli docker-ce-cli nodejs build-essential python-is-python3 python3-pip
+  awscli docker-ce-cli nodejs build-essential python-is-python3 \
+  python3-pip openjdk-8-jre wget
```

Comment:
which part needs jre?

I am not particularly happy with the jre installation.. but I guess I just should take it as it is (much) better than running on the ci test image..

Reply:
there is this lint written in java: https://github.com/ray-project/ray/blob/master/ci/lint/format.sh#L113